.github/workflows/unit-test.yml

```diff
@@ -146,12 +149,16 @@ jobs:
         run: npm install --ignore-scripts
       - name: Bootstrap Dependencies
         run: npx lerna bootstrap --no-ci --hoist --nohoist='zone.js' --nohoist='mocha' --nohoist='ts-mocha'
-      - name: Unit tests
+      - name: Unit tests (Full)
+        if: matrix.code-coverage
+        run: npm run test -- ${{ matrix.lerna-extra-args }}
+      - name: Unit tests (Delta)
+        if: ${{ !matrix.code-coverage }}
         run: npm run test:ci:changed -- ${{ matrix.lerna-extra-args }}
       - name: Build examples
         run: npm run compile:examples
       - name: Report Coverage
-        if: matrix.node == '14'
+        if: ${{ matrix.code-coverage || !cancelled()}}
```

Comment:
What is the purpose of `|| !cancelled()`? shouldn't it be `&&`to indicate code coverage AND not canceled? 

I always thought github actions will not attempt to run new jobs once the action is canceled.

Reply:
Nice cache, I update to `&& !cancelled()`

https://docs.github.com/en/actions/learn-github-actions/expressions#always

`!cancelled() === success() || failure()`

NYC will generate code coverage even when unit test is fail. So by using `&& |cancelled()` we will also collect those code coverage.

Reply:
Nice cache, I update to `&& !cancelled()`

https://docs.github.com/en/actions/learn-github-actions/expressions#always

`!cancelled() === success() || failure()`

NYC will generate code coverage even when unit test is fail. So by using `&& |cancelled()` we will also collect those code coverage.

Reply:
Thanks for clarifying 🥇